api/src/main/java/org/apache/brooklyn/api/mgmt/rebind/mementos/Memento.java

```diff
@@ -77,7 +79,8 @@
 
     public Collection<Object> getTags();
     
+    Map<String, Set<BrooklynObject>> getRelations();
```

Comment:
I'm surprised to see `BrooklynObject` reference in the memento, the existing practice is to store the IDs only. Not much of a difference in the end though since xstream will take care of it. May be we should do the same for the other references as well.